packages/plugin-ext/src/plugin/workspace.ts

```diff
@@ -56,6 +56,10 @@ export class WorkspaceExtImpl implements WorkspaceExt {
     private searchInWorkspaceEmitter: Emitter<{ result?: theia.TextSearchResult, searchId: number }> = new Emitter<{ result?: theia.TextSearchResult, searchId: number }>();
     protected workspaceSearchSequence: number = 0;
 
+    private _trusted: boolean = false;
```

Comment:
I believe we should persist the choice from the user, unfortunately on each startup the application will ask users which can become annoying.

Reply:
Great idea, I store the value now if the preference is set to ask only 'once'.